### PR TITLE
Custom css

### PR DESCRIPTION
**Pull request by Dramaturco**
Added the custom css and how to remove Tailwind sections to the documentation.

### CODE REVIEW

**Comment by Dramaturco** [comment]
Be aware that we haven't talked about what the "breadcrumbs" to the upper level of pages should be called. I called them "index" but I'm not sure if that's ideal. Also I'm not sure where to place them. I had them above the headline but the linter didn't like that.

File: docs/css/Component.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```css
+.headline {
+  font-size: 2rem;
+  font-family: sans-serif;
+  font-weight: bold;
+  color: coral;
+}
+```
+
+And then import and use it your component like this
```
Comment (tgolbs) [inline docs/css/Component.md:26]: *in

File: docs/css/RemovingTailwind.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Finally you have to remove the tailwind build module from the `nuxt.config.ts` file
+
+```typescript
+  /*
+   ** Nuxt.js dev-modules
+   */
+  buildModules: ['@nuxt/typescript-build', '@nuxtjs/tailwindcss'],
+```
+
+Keep in mind however, that the tailwind css classes are still applied to the existing components.
```
Comment (tgolbs) [inline docs/css/RemovingTailwind.md:21]: Remove ```tailwind.config.js``` as well.

**Review by tgolbs — CHANGES_REQUESTED** [review]
There are just two little issues. Rest looks great.

File: docs/css/Component.md (hunk 1 above)
Comment (Dramaturco) [reply docs/css/Component.md]: Check

File: docs/css/RemovingTailwind.md (hunk 2 above)
Comment (Dramaturco) [reply docs/css/RemovingTailwind.md]: Check

**Pull request merged**